From: Jan Engelhardt <jengelh@inai.de>
Date: 2021-05-12 17:36:02.536572543 +0200

Some dirs are not present in snapshots directly generated from git
(nor are they marked as a submodule). Patch it out, since we have
them as system libs.
---
 Makefile.am  |    4 ----
 configure.ac |    2 --
 2 files changed, 6 deletions(-)

Index: libcdata/Makefile.am
===================================================================
--- libcdata.orig/Makefile.am
+++ libcdata/Makefile.am
@@ -3,8 +3,6 @@ ACLOCAL_AMFLAGS = -I m4
 SUBDIRS = \
 	include \
 	common \
-	libcerror \
-	libcthreads \
 	libcdata \
 	po \
 	manuals \
@@ -54,8 +52,6 @@ lib: library
 
 library:
 	cd $(srcdir)/common && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libcerror && $(MAKE) $(AM_MAKEFLAGS)
-	cd $(srcdir)/libcthreads && $(MAKE) $(AM_MAKEFLAGS)
 	cd $(srcdir)/libcdata && $(MAKE) $(AM_MAKEFLAGS)
 	cd $(srcdir)/po && $(MAKE) $(AM_MAKEFLAGS)
 
Index: libcdata/configure.ac
===================================================================
--- libcdata.orig/configure.ac
+++ libcdata/configure.ac
@@ -94,8 +94,6 @@ dnl Generate Makefiles
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_FILES([include/Makefile])
 AC_CONFIG_FILES([common/Makefile])
-AC_CONFIG_FILES([libcerror/Makefile])
-AC_CONFIG_FILES([libcthreads/Makefile])
 AC_CONFIG_FILES([libcdata/Makefile])
 AC_CONFIG_FILES([po/Makefile.in])
 AC_CONFIG_FILES([po/Makevars])
